alluxio/configuration.rst

```diff
@@ -0,0 +1,32 @@
+**********************
+Alluxio Configuration
+**********************
+
+
+|fdd| provide Alluxio with the following configurations
+
+- UnderFS: HDFS is the default UnderFs 
+- Write Type: CACHE_THROUGH
```

Comment:
If `CACHE_THROUGH` is a value that should be used in code and/or configuration, perhaps it would be better to wrap it inside backticks to display it in a fixed-size font. The same should be applied for similar occurrences.